_includes/use-cases/use-case-consumption-execute-a-thirdpartyattributequery.md

```diff
@@ -0,0 +1,19 @@
+{{properties.description}}
+
+{% include properties_list.html %}
+
+This use-case is intended to search for a peer's relationship attribute that was created
```

Comment:
This use-case is intended to execute an incoming ThirdPartyRelationshipAttributeQuery (e.g. of a ReadAttributeRequestItem) which returns a list of matching [Relationship Attributes]({% link _docs_integrate/data-model-overview.md %}#relationshipattribute) of another contact.